hasura/metadata/databases/default/tables/public_profiles.yaml

```diff
@@ -77,4 +77,6 @@ update_permissions:
     filter:
       id:
         _eq: X-Hasura-User-Id
+    set:
+      id: x-hasura-User-Id
```

Comment:
lets make it consistent `X-Hasura-User-Id`,

think its also missing an update permission for `superadmin`here, permissions can be the same as `user` role

Reply:
ok i didn't change these but I can sync them up

Reply:
We can only customize the error message in the client side javascript. There is a LOT of this kind of stuff... every zod error is pretty tough too. Not sure I want to take this on right now

Reply:
> ok i didn't change these but I can sync them up


yeah np, all the permission settings was done by me during the start, probably was me that left that out